python/chatting_sample.py

```diff
@@ -0,0 +1,390 @@
+                return False
+
+        def set_error_dict(json_words):
+            for idx, word in enumerate(json_words):
+                if get_prosody_error("MissingBreak", word, json_words[idx-1]):
+                    error_dict["Missing break"].append(word)
+                elif get_prosody_error("UnexpectedBreak", word, json_words[idx-1]):
+                    error_dict["Unexpected break"].append(word)
+                elif get_prosody_error("Monotone", word, json_words[idx-1]):
+                    error_dict["Monotone"].append(word)
+
+        def get_error_message(error_types):
+            message = ""
+            for error_type in error_types:
+                if len(error_dict[error_type]) != 0:
+                    message += f"{error_type} count is {len(error_dict[error_type])}. near the word "
+                    message += f"{' '.join([word['Word'].strip() for word in error_dict[error_type]])}. "
+
+            return message
+
+        def get_report(json_words, mis_pronunciation_words):
+
+            set_error_dict(json_words)
+
+            origin_content = ""
+            if len(mis_pronunciation_words) != 0:
+                origin_content = "Accuracy report:"
+                for mis_word in mis_pronunciation_words:
+                    phone = "".join([syllable.syllable for syllable in mis_word.syllables])
+                    origin_content += f" word {mis_word.word}"
+                    origin_content += f" correct pronunciation is {mis_word.word},"
+                    origin_content += (
+                        f' your pronunciation is <phoneme alphabet="ipa" ph="{phone}">'
+                        f'{mis_word.word}'
+                        '</phoneme>. '
+                    )
+
+            if scores_dict["fluency score"] < 60 or scores_dict["prosody score"] < 60:
+                if scores_dict["fluency score"] < 60:
+                    origin_content += "Fluency "
+                if scores_dict["prosody score"] < 60:
+                    origin_content += "Prosody "
+                origin_content += "report: "
+                origin_content += get_error_message(["Missing break", "Unexpected break", "Monotone"])
+
+            tts(f'<speak version="1.0" xmlns="http://www.w3.org/2001/10/synthesis" xml:lang="en-US">\
+                    <voice name="en-US-JennyNeural">\
+                        {origin_content}\
+                    </voice>\
+                </speak>', "chat_report_output.wav")
+
+        def get_score_comment(scores_dict):
+            for score_key in scores_dict:
+                set_message_dict(scores_dict[score_key], score_key)
+
+            messages = ""
+            for message_key in message_dict:
+                if message_dict[message_key] != []:
+                    is_or_are = "is" if len(message_dict[message_key]) == 1 else "are"
+                    messages += f"{' and '.join(message_dict[message_key])} {is_or_are} {message_key}. "
+
+            tts((
```

Comment:
too many "and" in the score comment output.

Reply:
> too many "and" in the score comment output.

I changed "and" to "," just like in the studio

Reply:
Please add "and" for the last dimension.